openmetadata-ui/src/main/resources/ui/src/components/ContainerDetail/ContainerDataModel/ContainerDataModel.tsx

```diff
@@ -0,0 +1,284 @@
+            />
+          )}
+        </Space>
+      );
+    };
+
+  const renderContainerColumnTags: CellRendered<Column, 'tags'> = (
+    tags,
+    record: Column
+  ) => {
+    const isSelectedField = editContainerColumnTags?.name === record.name;
+    const styleFlag = isSelectedField || !isEmpty(tags);
+
+    return (
+      <>
+        {isReadOnly ? (
+          <Space wrap>
+            <TagsViewer sizeCap={-1} tags={tags || []} />
+          </Space>
+        ) : (
+          <Space
+            align={styleFlag ? 'start' : 'center'}
+            className="justify-between"
+            data-testid="tags-wrapper"
+            direction={styleFlag ? 'vertical' : 'horizontal'}
+            onClick={() => handleAddTagClick(record)}>
+            <TagsContainer
+              editable={isSelectedField}
+              isLoading={isTagLoading && isSelectedField}
+              selectedTags={tags || []}
+              showAddTagButton={hasTagEditAccess}
+              size="small"
+              tagList={tagList}
+              type="label"
+              onCancel={() => setEditContainerColumnTags(undefined)}
+              onSelectionChange={handleFieldTagsChange}
+            />
+          </Space>
+        )}
+      </>
+    );
+  };
+
+  const columns: ColumnsType<Column> = useMemo(
+    () => [
+      {
+        title: t('label.name'),
+        dataIndex: 'name',
+        key: 'name',
+        accessor: 'name',
+        width: 300,
+        render: (_, record: Column) => (
+          <Popover
+            destroyTooltipOnHide
+            content={getEntityName(record)}
+            trigger="hover">
+            <Typography.Text>{getEntityName(record)}</Typography.Text>
+          </Popover>
+        ),
+      },
+      {
+        title: t('label.type'),
+        dataIndex: 'dataTypeDisplay',
+        key: 'dataTypeDisplay',
+        accessor: 'dataTypeDisplay',
+        ellipsis: true,
+        width: 220,
+        render: (dataTypeDisplay: Column['dataTypeDisplay']) => {
+          return (
+            <Popover
+              destroyTooltipOnHide
+              content={toLower(dataTypeDisplay)}
+              overlayInnerStyle={{
+                maxWidth: '420px',
+                overflowWrap: 'break-word',
+                textAlign: 'center',
+              }}
+              trigger="hover">
+              <Typography.Text ellipsis className="cursor-pointer">
+                {dataTypeDisplay}
+              </Typography.Text>
+            </Popover>
+          );
+        },
+      },
+      {
+        title: t('label.description'),
+        dataIndex: 'description',
+        key: 'description',
+        accessor: 'description',
+        render: renderContainerColumnDescription,
+      },
+      {
+        title: t('label.tag-plural'),
+        dataIndex: 'tags',
+        key: 'tags',
+        accessor: 'tags',
+        width: 350,
+        render: renderContainerColumnTags,
+      },
+    ],
+    [
+      dataModel,
```

Comment:
I don't think we need this dependency here?

Reply:
Yes, it should not be there, I will remove it.